idl/cdc_topology_description.idl.hh

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace cdc {
+
+class stream_id final {
+    int64_t _first;
+    int64_t _second;
+};
+
+class topology_description final {
```

Comment:
Are you sure we will never want to extend it? final is great for high-use objects where every byte matters, but this isn't such a case.

Reply:
Dunno. Removed `final`